docs/xcm/building-with-xcm/xc-assets-withdraw.md

```diff
@@ -0,0 +1,38 @@
+
+# Withdraw Assets
+
+In polkadot ecosystem there are two ways to transfer assets between remote chains: teleportation and reserve transfer. The first is essentially burning thing on one side and minting it on the other, so that the total balance before and after the teleportation remains the same.
+
+The second way to transfer assets uses cross chain assets backed by some assets on the sending side. Assets to be transferred are locked on the sender's side but are then said to be belonging to the remote side. Depending on the business logic, remote side now owning the assets can mint some derivatives to reflect that ownership. That way, original assets never actually leave the chain, but their ownership does. Such transfer can be done using [`assets_reserve_transfer`](xc-reserve-transfer) API.
+
+Afterwards, when remote chain wishes to redeem some original assets it can use `assets_withdraw` to perform the reverse operation.
```

Comment:
I find this a bit misleading since the remote chain doesn't have to use these precompiles or even support precompiles - it's more about executing a certain sequence of XCM instructions.

Perhaps it can be worded in a way that doesn't imply precompile usage.